tools/report-converter/codechecker_report_converter/coccinelle/output_parser.py

```diff
@@ -0,0 +1,65 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+
+import logging
+import os
+import re
+
+from ..output_parser import BaseParser, Message
+LOG = logging.getLogger('ReportConverter')
+
+
+class CoccinelleParser(BaseParser):
+    """
+    Parser for Coccinelle Output
+    """
+
+    def __init__(self, analyzer_result):
+        super(CoccinelleParser, self).__init__()
+
+        self.analyzer_result = analyzer_result
+
+        self.message_line_re = re.compile(
+            # File path followed by a ':'.
+            r'^(?P<path>[\S ]+?):'
+            # Line number followed by a ':'.
+            r'(?P<line>\d+?):'
+            r'(?P<column>(\d+?)-(\d+?):)'
```

Comment:
This will match the following string: `13-14:`. So it will be a string and it also includes the `:`. The column number below when creating a `Message` object should be a number. So it will fail because you pass a string.

I recommend to use the following regex:
```suggestion
            r'(?P<column>(\d+?))-(\d+?):'
```


Please also add a comment above this line that this is a range of column numbers and we use only the start of this range.